decomp/vis/uds_vis.py

```diff
@@ -0,0 +1,786 @@
+from typing import List, Dict
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import networkx as nx
+import plotly.graph_objs as go
+import numpy as np
+import matplotlib
+from matplotlib import transforms
+import json
+import jsonpickle
+
+from .. import UDSCorpus
+from ..semantics.uds import UDSGraph
+from decomp.vis.ontology import NODE_ONTOLOGY, EDGE_ONTOLOGY
+
+class StringList:
+    def __init__(self, text):
+        self.text_list = text.split(" ")
+
+    def index(self, item):
+        try:
+            return self.text_list.index(item)
+        except ValueError:
+            return 1000
+
+    def __str__(self):
+        return " ".join(self.text_list) 
+
+
+class UDSVisualization:
+    def __init__(self,
+                 graph: UDSGraph, 
+                 add_span_edges: bool = True,
+                 add_syntax_edges: bool = False,
+                 from_prediction: bool = False,
+                 sentence: str = None,
+                 syntax_y: float = 0.0,
+                 semantics_y: float = 10.0,
+                 node_offset: float = 7.0,
+                 width: float = 1000,
+                 height: float = 400) -> None:
```

Comment:
Conform to type-hinting style in rest of package, which does not specify output type for `__init__`.